src/ui/pages/equipment/asset/AssetGeneral.js

```diff
@@ -11,7 +12,6 @@ class AssetGeneral extends Component {
 
         return (
             <div style={{width: "100%", marginTop: 0}}>
-
```

Comment:
This change is not relevant to the PR. We should avoid having blank line additions/removals in unrelated lines/files.